src/stan_math_backend/Transform_Mir.ml

```diff
@@ -6,53 +6,42 @@ let use_opencl = ref false
 let opencl_triggers =
   String.Map.of_alist_exn
     [ ( "normal_id_glm_lpdf"
-      , ( [0; 1]
-        , [ (* Array of conditions under which to move to OpenCL *)
-            ([1], (* Argument 1 is data *)
-                  [(1, UnsizedType.UMatrix)])
-          (* Argument 1 is a matrix *)
-           ] ) )
-    ; ( "bernoulli_logit_glm_lpmf"
-      , ([0; 1], [([1], [(1, UnsizedType.UMatrix)])]) )
-    ; ( "categorical_logit_glm_lpmf"
-      , ([0; 1], [([1], [(1, UnsizedType.UMatrix)])]) )
-    ; ( "neg_binomial_2_log_glm_lpmf"
-      , ([0; 1], [([1], [(1, UnsizedType.UMatrix)])]) )
-    ; ( "ordered_logistic_glm_lpmf"
-      , ([0; 1], [([1], [(1, UnsizedType.UMatrix)])]) )
-    ; ("poisson_log_glm_lpmf", ([0; 1], [([1], [(1, UnsizedType.UMatrix)])]))
-    ]
+      , [ (* Array of conditions under which to move to OpenCL *)
+          [(1, UnsizedType.UMatrix)]
+        (* Argument 1 is a matrix *)
+         ] )
+    ; ("bernoulli_logit_glm_lpmf", [[(1, UnsizedType.UMatrix)]])
```

Comment:
isn't the 1st argument of these glms always a matrix?

Reply:
It can also be `row_vector`. here are all 32 GLM sigs:

```stan
real bernoulli_logit_glm_lpmf(int, matrix, real, vector)
real bernoulli_logit_glm_lpmf(int, matrix, vector, vector)
real bernoulli_logit_glm_lpmf(int[], row_vector, real, vector)
real bernoulli_logit_glm_lpmf(int[], row_vector, vector, vector)
real bernoulli_logit_glm_lpmf(int[], matrix, real, vector)
real bernoulli_logit_glm_lpmf(int[], matrix, vector, vector)
real categorical_logit_glm_lpmf(int, row_vector, vector, matrix)
real categorical_logit_glm_lpmf(int, matrix, vector, matrix)
real categorical_logit_glm_lpmf(int[], row_vector, vector, matrix)
real categorical_logit_glm_lpmf(int[], matrix, vector, matrix)
real neg_binomial_2_log_glm_lpmf(int, matrix, real, vector, real)
real neg_binomial_2_log_glm_lpmf(int, matrix, vector, vector, real)
real neg_binomial_2_log_glm_lpmf(int[], row_vector, real, vector, real)
real neg_binomial_2_log_glm_lpmf(int[], row_vector, vector, vector, real)
real neg_binomial_2_log_glm_lpmf(int[], matrix, real, vector, real)
real neg_binomial_2_log_glm_lpmf(int[], matrix, vector, vector, real)
real normal_id_glm_lpdf(real, matrix, real, vector, vector)
real normal_id_glm_lpdf(real, matrix, vector, vector, vector)
real normal_id_glm_lpdf(vector, row_vector, real, vector, vector)
real normal_id_glm_lpdf(vector, row_vector, vector, vector, vector)
real normal_id_glm_lpdf(vector, matrix, real, vector, real)
real normal_id_glm_lpdf(vector, matrix, vector, vector, real)
real ordered_logistic_glm_lpmf(int, row_vector, vector, vector)
real ordered_logistic_glm_lpmf(int, matrix, vector, vector)
real ordered_logistic_glm_lpmf(int[], row_vector, vector, vector)
real ordered_logistic_glm_lpmf(int[], matrix, vector, vector)
real poisson_log_glm_lpmf(int, matrix, real, vector)
real poisson_log_glm_lpmf(int, matrix, vector, vector)
real poisson_log_glm_lpmf(int[], row_vector, real, vector)
real poisson_log_glm_lpmf(int[], row_vector, vector, vector)
real poisson_log_glm_lpmf(int[], matrix, real, vector)
real poisson_log_glm_lpmf(int[], matrix, vector, vector)
```

Reply:
This was added fairly recently https://github.com/stan-dev/stanc3/pull/415

Reply:
Ah okay, I was looking at an older copy of the manual. thanks!